stubs/ldap3/ldap3/protocol/rfc4511.pyi

```diff
@@ -1,32 +1,26 @@
-from typing import Any as _Any
-
-# Enable when pyasn1 gets stubs:
-# from pyasn1.type.univ import Boolean, Choice, Enumerated, Integer, Null, OctetString, Sequence, SequenceOf, SetOf
-Boolean = _Any
-Choice = _Any
-Enumerated = _Any
-Integer = _Any
-Null = _Any
-OctetString = _Any
-Sequence = _Any
-SequenceOf = _Any
-SetOf = _Any
-
-LDAP_MAX_INT: int
-MAXINT: _Any
-rangeInt0ToMaxConstraint: _Any
-rangeInt1To127Constraint: _Any
-size1ToMaxConstraint: _Any
-responseValueConstraint: _Any
-numericOIDConstraint: _Any
-distinguishedNameConstraint: _Any
-nameComponentConstraint: _Any
-attributeDescriptionConstraint: _Any
-uriConstraint: _Any
-attributeSelectorConstraint: _Any
+from typing_extensions import Final as _Final
+
+from pyasn1.type.constraint import ConstraintsIntersection, SingleValueConstraint, ValueRangeConstraint, ValueSizeConstraint
+from pyasn1.type.namedtype import NamedTypes
+from pyasn1.type.namedval import NamedValues
+from pyasn1.type.tag import TagSet
+from pyasn1.type.univ import Boolean, Choice, Enumerated, Integer, Null, OctetString, Sequence, SequenceOf, SetOf
+
+LDAP_MAX_INT: _Final = 2147483647  # noqa: Y015  # name conflict
```

Comment:
We just disabled Y015 entirely in #9490!

```suggestion
LDAP_MAX_INT: _Final = 2147483647
```

Reply:
I can remove the comment (explained above), but gotta keep the noqa https://github.com/PyCQA/flake8-pyi/issues/316

Reply:
Yeah, but we now ignore all Y015 errors in typeshed following #9490: https://github.com/python/typeshed/blob/ab1921a737456688b93a43fc7c38021927d0f13c/.flake8#L33

Reply:
Confirmed locally that flake8 still passes with the `# noqa` removed, due to the global ignore.

I guess flake8-noqa isn't complaining because there _is_ a genuine Y015 violation on this line, so the `# noqa` isn't actually _unused_ -- it's just "redundant", since it's already covered by the per-file ignore in the config file.

Reply:
Indeed I pulled main and it works now.

Interesting, I wonder if that's something they could improve or if it's a hard limitation.

Reply:
I haven't looked into flake8-noqa does its magic -- no idea really! 😀

Reply:
Looks like a known issue: https://github.com/plinss/flake8-noqa/issues/12
I had already upvoted it and didn't even remember ^^"